ingest/validation/validate_metadata.py

```diff
@@ -1163,21 +1222,30 @@ def validate_collected_ontology_data(metadata, convention):
                             "error",
                             "ontology",
                             error_msg,
-                            metadata.ontology[property_name][
+                            associated_info=metadata.ontology[property_name][
                                 (ontology_id, ontology_label)
                             ],
+                            issue_name="content:metadata:label-not-match-id",
                         )
             except ValueError as valueError:
                 metadata.store_validation_issue(
                     "error",
                     "ontology",
                     valueError.args[0],
-                    metadata.ontology[property_name][(ontology_id, ontology_label)],
+                    associated_info=metadata.ontology[property_name][
+                        (ontology_id, ontology_label)
+                    ],
+                    issue_name="content:metadata:ontology-label-lookup",
                 )
             except requests.exceptions.RequestException as err:
                 error_msg = f"External service outage connecting to {ontology_urls} when querying {ontology_id}:{ontology_label}: {err}"
                 dev_logger.exception(error_msg)
-                metadata.store_validation_issue("error", "ontology", error_msg),
+                metadata.store_validation_issue(
+                    "error",
+                    "ontology",
+                    error_msg,
+                    issue_name="external_server:request-exception",
+                ),
```

Comment:
I don't think this is really a validation issue, per se.  It certainly _relates_ to validation, but it's more of an application error than a user error like all other validation issues.

So I'd log this to Sentry (and maybe Mixpanel too) as a handled error where we want easy observability, but logging it as a validation error strikes me as a category mistake.

```suggestion
```